docs/user/subprojects.rst

```diff
@@ -1,43 +1,67 @@
-Subprojects
-===========
+Subprojects: host multiple projects on a single domain
+======================================================
 
-Projects can be configured in a nested manner, by configuring a project as a
-*subproject* of another project. This allows for documentation projects to share
-a search index and a namespace or custom domain, but still be maintained
-independently.
+In this article, you can learn more about how several documentation projects can be combined and presented to the reader on the same website.
 
-For example, a parent project, ``Foo`` is set up with a subproject, ``Bar``. The
-documentation for ``Foo`` will be available at:
+Read the Docs projects can be configured to make other *subprojects* available on the website of the *main project*.
+This allows for documentation projects to share a search index and a namespace or custom domain,
+but still be maintained independently.
 
-https://foo.readthedocs.io/en/latest/
+This is useful for:
 
-The documentation for ``Bar`` will be available under this same path:
+* Organizations that need all their projects visible in one documentation portal or landing page
+* Projects that document and release several packages or extensions
+* Organizations or projects that want to have a common search function for several sets of documentation
 
-https://foo.readthedocs.io/projects/bar/en/latest/
+For a main project ``example-project``, a subproject ``example-project-plugin`` can be made available as follows:
```

Comment:
This sentence is hard to read for me. I'm not sure if "can be made available" is what confuses me, there is a word missing or something else 🙃  --maybe I'm just tired.

Reply:
Let's fix it -- being tired is a good benchmark for readability :+1:

Reply:
Yea, `can be made available` is super passive voice. `Is activated` or similar would be active voice.